Automattic/jetpack
# AI Image: Empty state for carrousel

Ref: https://github.com/orgs/Automattic/projects/667/views/23?pane=issue&itemId=66869234

## Proposed changes:

* Move all components to `/components` folder.
* Add empty state using `AiIcon` for `carrousel`.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Open your editor.
* Add image block.
* Go to `Select Image > Generate with AI`.
* You should see the modal opening with empty state.
* Test `Featured Image` to avoid some regression.

#### Demo

![CleanShot 2024-06-24 at 10 28 27](https://github.com/Automattic/jetpack/assets/1663717/671edcb1-ffe9-4d8b-a8dd-4cbaea9337f7)
